framework/src/main/java/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -3793,22 +3793,22 @@ protected boolean checkMethodReferenceAsOverride(
     // enclosing method.
     // That is handled separately in method receiver check.
 
-    // The type of the expression or type use, <expression>::method or <type use>::method.
-    ExpressionTree qualifierExpression = memberReferenceTree.getQualifierExpression();
+    // preColonTree is an expression or type use, <expression>::method or <type use>::method.
```

Comment:
I prefer `qualifierExpression` because then it is easier to match the code to the JLS or javac.

Reply:
It doesn't help with the JLS, because "qualifier expression" never appears in the JLS.  Therefore, I found it confusing.  "Qualiifer expression" is a term made up by javac, it is not necessarily an expression, and it has nothing to do with type qualifiers which is what the Checker Framework is about.

Reply:
I guess I'm just used to the javac naming convention.  I can adjust to `preColonTree`.